src/main.rs

```diff
@@ -13,10 +13,18 @@ use std::process::ExitCode;
 use clap::Parser;
 
 pub(crate) mod cli;
+pub(crate) mod executor;
 
 use cli::Cli;
 use console::style;
 
+#[cfg(not(target_env = "msvc"))]
+use tikv_jemallocator::Jemalloc;
+
+#[cfg(not(target_env = "msvc"))]
+#[global_allocator]
+static GLOBAL: Jemalloc = Jemalloc;
+
```

Comment:
I don't think we should include this change as part of this PR. Changing the global allocator can have a non-trivial cost for file size (jemalloc was actually *removed* from rust because of this some time ago) and is best investigated separately from the rest of the changes here.

Reply:
Was primarily using Jemalloc for multi-threading purposes, but since that isn't necessary anymore, I should remove this too.

Reply:
I believed Jemalloc was needed even *after* removing the require bundling; probably because I hadn't properly read the logic in the past couple days before creating this PR. Apologies.

Reply:
Resolved as per https://github.com/lune-org/lune/pull/140/commits/6f4b2f4c313c74e9da6cef614a61de5a3cb1c5a8.